Machine Learning/IT Operations Recipes/service_response_change/data/apache_logs.log

```diff
@@ -12703,7 +12703,7 @@
 73.6.71.236 - - [21/Apr/2017:15:48:45 +0000] "GET /misc/sample.log HTTP/1.1" 200 54306753.0 "http://www.semicomplete.com/" "Mozilla/5.0 (Windows NT 5.1) AppleWebKit/537.17 (KHTML, like Gecko) Chrome/24.0.1312.56 Safari/537.17"
 180.122.254.219 - - [21/Apr/2017:15:48:46 +0000] "GET /presentations/logstash-puppetconf-2013/images/logstash-web.png HTTP/1.1" 200 264165.0 "http://semicomplete.com/presentations/logstash-puppetconf-2013/" "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10.6; en-US; rv:1.9.2) Gecko/20100115 Firefox/3.6 (FlipboardProxy/2.0; +http://flipboard.com/browserproxy)"
 218.168.171.72 - - [21/Apr/2017:15:48:52 +0000] "GET /blog/2007/May/20 HTTP/1.1" 200 8583.0 "-" "Mozilla/4.0 (MSIE 6.0; Windows NT 5.0)"
-52.153.198.157 - - [21/Apr/2017:15:48:52 +0000] "GET /blog/geekery/jquery-mobile-full-height-content.html HTTP/1.1" 200 9571.0 "http://stackoverflow.com/questions/17083473/phonegap-get-screen-height-of-working-area-screen-less-the-status-bar" "DoCoMo/2.0 N905i(c100;TB;W24H16) (compatible; Googlebot-Mobile/2.1; +http://www.google.com/bot.html)"
```

Comment:
@ypid-geberit Why are you only changing `http` to `https` for [stackoverflow.com](https://stackoverflow.com)?

There are other URLs in the logs. Example: `http://semicomplete.com/presentations/logstash-scale11x/`

Reply:
The URLs should still work. So I only changed URLs which I know support HTTPS. That was 2017. In 2021, we are at 85 % (https://letsencrypt.org/stats/). So I still cannot just upgrade all of the HTTP URLs. But since that commit I made progress and found the ideal tool to do such mechanical edits, HTTPS Everywhere, see https://github.com/EFForg/https-everywhere/pull/19869. The setup is a bit involved and I don’t really invest more time into this PR right now before I see my work actually getting merged. I can do a full update in a second round if you are interested.